packages/datadog-plugin-connect/test/index.spec.js

```diff
@@ -504,82 +505,234 @@ describe('Plugin', () => {
           connect = require(`../../../versions/connect@${version}`).get()
         })
 
-        it('should be configured with the correct service name', done => {
-          const app = connect()
+        describe('with middleware disabled', () => {
```

Comment:
This tests more than that.

Reply:
good point. I've updated these tests so that the testing of the other parts of the config is separate from testing the middleware functionality